packages/cross-runtime/services/WordCounter.service.ts

```
@@ -0,0 +1,9 @@
+export class WordCounterService {
+  public static countWords(text: string): number {
+    return text.trim().split(/\s+/).filter(Boolean).length;
+  }
+
+  public static countMinutesToRead(text: string): number {
+    return Math.max(1, Math.ceil(this.countWords(text) / 200));
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)